langgraph/demo.py

```
@@ -0,0 +1,151 @@
+  - Always specify both filename and projectname parameters
+  - Read key files like README.md, main entry points, configuration files, or any files that reference the target service
+  - Don't read every file - be strategic and focus on files that will provide the most relevant context
+
+  Return the exact toolcall you are making, alongside the passed parameters to those tools as part of the response. 
+""",
+    name="repo_assistant"
+)
+
+def list_team_users(team_name: str):
+    """List out the team members from a certain specific team, for example Frontend Engineering"""
+    team_members_query_result = directory_db_query(f"SELECT users.name FROM users JOIN teams ON users.team_id = teams.id WHERE teams.name = \'{team_name}\'")
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)